src/MongoDB.Driver.Core/Core/Operations/ListCollectionsUsingQueryOperation.cs

```diff
@@ -144,7 +144,7 @@ private IEnumerable<BsonDocument> NormalizeQueryResponse(IEnumerable<BsonDocumen
                 if (name.StartsWith(prefix))
                 {
                     var collectionName = name.Substring(prefix.Length);
-                    if (!collectionName.Contains('$'))
+                    if (collectionName != "system.indexes" && !collectionName.Contains('$'))
```

Comment:
This *could* be considered a backward breaking change, *but*... this operation is *supposed* return the exact same thing as the listCollections command (this operation is emulating that command).